tests/e2e/reborn_webui_harness.py

```
@@ -34,6 +38,152 @@
+        ) from error
+    if max_bytes <= 0:
+        raise ValueError(
+            "IRONCLAW_E2E_ARTIFACT_MAX_BYTES must be a positive integer"
+        )
+    return max_bytes
+
+
+class _ArtifactContext:
+    """Browser context that persists diagnostics when CI requests them."""
+
+    def __init__(
```

Ruff (0.16.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)